test/inspector/test-inspector-stops-no-file.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+require('../common');
+const spawn = require('child_process').spawn;
+
+const child = spawn(process.execPath,
+      [ '--inspect', 'no-such-script.js' ],
+      { 'stdio': 'inherit' });
+
+function exitHandler() {
+  child.kill();
+}
+
+process.on('exit', exitHandler);
+process.on('SIGINT', exitHandler);
+process.on('SIGTERM', exitHandler);
```

Comment:
Installing listeners for SIGINT and SIGTERM will override the default behavior of terminating when the signal is received.  Call `process.exit()` in the callback.